client/src/components/GameInvitation.js

```diff
@@ -0,0 +1,64 @@
+/*
+Component for inviting friends to a new game.
+*/
+
+import React, { useState, useEffect } from "react";
```

Comment:
```suggestion
import React, { useState } from "react";
```
At the moment useEffect is not being used